glitter_news/admin.py

```diff
@@ -35,7 +35,9 @@ def get_fieldsets(self, request, obj=None):
 
         fieldsets = (
             ('Post', {
-                'fields': ('title', 'category', 'author', 'date', 'image', 'summary', 'tags')
+                'fields': (
+                    'title', 'category', 'is_sticky', 'author', 'date', 'image', 'summary', 'tags',
```

Comment:
Could we move `is_sticky` to the advanced options?

Feels like an optional ignorable thing.